pkg/telemetry/tracing.go

```diff
@@ -1,10 +1,22 @@
 package telemetry
 
 import (
+	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/codes"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 	"go.opentelemetry.io/otel/trace"
 )
 
+func MustNewTracerProvider() trace.TracerProvider {
```

Comment:
Why are there tracing changes in a logger focused PR?

Reply:
Yeah, good point. I will revert this stuff and add it to the tracer PR. Thanks!